extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -229,13 +229,6 @@ export class AmpStoryEmbeddedComponent {
         this.setState_(EmbeddedComponentState.HIDDEN, null /** component */);
         break;
       case EmbeddedComponentState.FOCUSED:
-        if (this.state_ !== EmbeddedComponentState.HIDDEN) {
```

Comment:
Intentional?

Reply:
Yes, we used to have 3 states (these ones + `EXPANDED`) but now we just have 2, so this check is no longer necessary.

It used to warn when we went from `EXPANDED` to `FOCUSED` which didn't make sense.

Reply:
Got it, opportunistic cleanup.

Thanks!